mage_ai/frontend/components/datasets/FeatureProfiles.tsx

```diff
@@ -193,15 +172,27 @@ function FeatureProfiles({
         </Text>
       </HeaderStyle>
       <BodyStyle>
-        {features.map((feature, idx) => (
-          <FeatureProfileStyle key={idx}>
-            <FeatureProfile
-              feature={feature}
-              featureSet={featureSet}
-              statistics={statistics}
-            />
-          </FeatureProfileStyle>
-        ))}
+        <Flex>
```

Comment:
Here as well. `Flex` might work, but in terms of code maintainability, other devs might wonder why you chose this versus `FlexContainer`.